CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased](https://github.com/ckan/ckanext-dcat/compare/v1.1.0...HEAD)
 
 * TBD
+* Introduce new interface method `after_parsing`
```

Comment:
```suggestion
* Introduce new interface method `after_parsing` (#196)
```

Reply:
@amercader Thanks for the fast review and the suggested change. I will push a new commit updating the changelog with some additional minor changes, e.g. changing the comparison from the latest 1.1.1 to the current master.